PopulationDensity.ipynb

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
<p>Committing the output of this would be nice, so that you'd get an idea of what the code does without actually running it</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laitalaj/railanalysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>